deployment/terraform/assets/cluster.tf

```diff
@@ -269,6 +277,13 @@ resource "aws_iam_user_policy" "s3userpolicy" {
 EOF
 }
 
+resource "aws_db_subnet_group" "redis" {
```

Comment:
The word "db" sounds off here. Can we change to something better?

Additionally, we need index specific tags here like:

```
 tags = {
    Name = "${var.cluster_name}-proxy-${count.index}"
  }
```

Reply:
That's the name of the AWS resource, I can't change that.